src/Microsoft.ML.Data/Scorers/BinaryClassifierScorer.cs

```diff
@@ -290,7 +289,7 @@ private static ColumnType GetPredColType(ColumnType scoreType, ISchemaBoundRowMa
             var labelNameBindableMapper = mapper.Bindable as MultiClassClassifierScorer.LabelNameBindableMapper;
             if (labelNameBindableMapper == null)
                 return BoolType.Instance;
-            return new KeyType(DataKind.U4, 0, labelNameBindableMapper.Type.VectorSize);
+            return new KeyType(DataKind.U4, 0, labelNameBindableMapper.Type.Size);
```

Comment:
why not GetVectorSize?

Reply:
Better to use the actual real property on `VectorType` directly if we can.